files/es/glossary/base64/index.md

```diff
@@ -17,68 +17,59 @@ La función `atob()` decodifica una cadena de datos que ha sido codificada usand
+    - trabajar con otras codificaciones Unicode diferentes de las [`DOMString`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String)s UTF-16 por defecto de JavaScript.
 
 **Herramientas**
 
-* [Reescribir atob() y btoa() usando TypedArrays y UTF-8](https://developer.mozilla.org/es/docs/glossary/base64#solution_.232_.e2.80.93_rewriting_atob%28%29_and_btoa%28%29_using_typedarrays_and_utf-8)
-* [`StringView`, una representación parecida a C de cadenas basadas en arrays tipados](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Typed_arrays/StringView "This is a link to an unwritten page")
+- [Reescribir atob() y btoa() usando TypedArrays y UTF-8](https://developer.mozilla.org/es/docs/glossary/base64#solution_.232_.e2.80.93_rewriting_atob%28%29_and_btoa%28%29_using_typedarrays_and_utf-8)
+- [`StringView`, una representación parecida a C de cadenas basadas en arrays tipados](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Typed_arrays/StringView)
 
 **Temas relacionados**
 
-* [`ArrayBuffer`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/ArrayBuffer)
-* [Vectores o arrays tipados](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Typed_arrays)
-* [ArrayBufferView](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/TypedArray)
-* [`Uint8Array`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Uint8Array)
-* [`StringView`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Typed_arrays/StringView "/en-US/docs/Web/JavaScript/Typed_arrays/StringView")[, una representación parecida a C de cadenas basadas en arrays tipados](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Typed_arrays/StringView "This is a link to an unwritten page")
-* [`DOMString`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String "/en-US/docs/Web/API/DOMString")
-* [`URI`](https://developer.mozilla.org/en-US/docs/Glossary/URI "/en-US/docs/URI")
-* [`encodeURI()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/encodeURI "/en-US/docs/Web/JavaScript/Reference/Global_Objects/encodeURI")
-    </td>
-  </tr>
-<table>
+- [`ArrayBuffer`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/ArrayBuffer)
+- [Vectores o arrays tipados](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Typed_arrays)
+- [ArrayBufferView](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/TypedArray)
+- [`Uint8Array`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Uint8Array)
+- [`StringView`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Typed_arrays/StringView)[, una representación parecida a C de cadenas basadas en arrays tipados](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Typed_arrays/StringView)
```

Comment:
Unrelated to the linting changes, but we have a link label here 

```
[, una representación parecida a C de cadenas basadas en arrays tipados]
```

I think this might be misuse of a label. Looks like the only use of this kind of label